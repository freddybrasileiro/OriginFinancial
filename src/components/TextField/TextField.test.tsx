import * as React from 'react';
import { shallow } from 'enzyme';
import { TextField } from './index';

describe('TextField', () => {
  it('Should render the TextField component', () => {
    expect(shallow(<TextField adornment="$" placeholder="" label="Total amount" onChange={()=>"{}"} value={25000} defaultValue={25000} />)).toMatchSnapshot();
  });
});
