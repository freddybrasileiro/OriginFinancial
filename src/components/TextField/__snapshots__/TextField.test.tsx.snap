// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextField Should render the TextField component 1`] = `ShallowWrapper {}`;
